client/extensions/woocommerce/state/ui/shipping/zones/test/selectors.js

```diff
@@ -41,12 +41,42 @@ describe( 'selectors', () => {
 			const state = {
```

Comment:
Yeah, I get it, but the tests that need that kind of setup are usually the simple "check that the selector is actually using the `siteId` property to get the bransh state", no need to do a helper for that because the state tree is not so big.

If you're testing something that requires orders/products/whatever from 2 different sites, then that's probably because you're testing a multi-site UI (which we don't have yet). In that case, maybe if would be worth it to use the `fixture` file you suggested, or to rework the `createState` so it can accomodate tat case without making the normal tests bigger.